content/pages/download.md

```diff
@@ -232,27 +232,27 @@ However, your (or someone else's) government may be able to find out who you are
 
 ## Verifying Signatures
 
-        Download the [Freenet Project signing keys][url_keyring] and import them into your keyring:
+Download the [Freenet Project signing keys][url_keyring] and import them into your keyring:
 
-        pub   2048R/7AA9C2A3 2013-04-29
+        pub   2048R/0xEAC5EBF07AA9C2A3 2013-04-29
```

Comment:
2048R may be a 1.4/2.0/2.1 difference, but the 0x is intentional, it's a keyid formatting option when printing keys with gpg. If the 0x isn't copied along with the rest of the ID, some keyservers will act like they can't find the key, particularly sks:

This works:
   
[https://sks-keyservers.net/pks/lookup?op=vindex&search=0xB67C19E817A8D846](https://sks-keyservers.net/pks/lookup?op=vindex&search=0xB67C19E817A8D846)

This doesn't: 
    
[https://sks-keyservers.net/pks/lookup?op=vindex&search=B67C19E817A8D846](https://sks-keyservers.net/pks/lookup?op=vindex&search=B67C19E817A8D846)

Since people commonly complain that the keys aren't on the keyservers, I figured that may be one reason they can't find them.